codemods/landingPageHero.js

```diff
@@ -0,0 +1,44 @@
+const visit = require('unist-util-visit');
+const { mdxBlockElement } = require('./utils/mdxast-builder');
+
+const containsImage = (node) => {
+  if (node.type === 'image') {
+    return true;
+  }
+
+  return node.children && node.children.some(containsImage);
+};
+
+// This codemod assumes it is run after the landing page tile grid has been run
+const landingPageHero = () => (tree, file) => {
+  if (file.data.doc.type !== 'landing_page') {
+    return;
+  }
+
+  visit(tree, 'root', (root) => {
+    const frontmatterIdx = 0; // frontmatter always comes first
```

Comment:
If frontmatter always comes first, is there any advantage to just writing `.slice(1, gridIdx)` below? I'm not against it...just curious if we gain anything by attaching some context to `0`.

Reply:
Not sure. `1` felt like a magic number to me, hence why I opted for the variable. That being said, this was probably more of a long-term strategy than it needs to be. This codemod really only needs to survive until GA. I'll go ahead and switch to use `1`. If this were something that stuck around for the lifetime of the application, I'd probably opt for this so that updating this 6 months from now was obvious was that `1` was for 😄